tapis-project/tapis-ui
# task/TUI-74 - dashboard component

## Overview: ##

Dashboard component

## Related Github Issues: ##

* [TUI-74](https://github.com/tapis-project/tapis-ui/issues/74)

## Summary of Changes: ##

## Testing Steps: ##
1. Log in with cicsvc
2. You should see a summary of tapis objects in the dashboard

## UI Photos:

![image](https://user-images.githubusercontent.com/17181582/124980633-41947b00-dffa-11eb-9cda-5490a5cdbf34.png)


## Notes: ##
